switch to rattler build

evonreis opened the pull request:
…nda-forge-pinning 2025.08.06.23.06.56

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

erikvonreis committed:
MNT: Re-rendered with conda-build 25.7.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.08.06.23.06.56

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but couldn't find any.
Please ping the 'conda-forge/core' team (using the `@` notation in a comment) if you believe this is a bug.

erikvonreis committed:
Add recipe.yaml

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found some lint.

Here's what I've got...


For **recipe/recipe.yaml**:

 * ❌ The recipe must have some tests.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/16815961438](https://github.com/conda-forge/conda-forge-webservices/actions/runs/16815961438). Examine the logs at this URL for more detail.</sub>

erikvonreis committed:
Patch compatible version of has_single_bit

erikvonreis committed:
Have to add patch

erikvonreis committed:
rename a header that had the same name as another header

fSeries.hh and FSeries.hh differed only by case
which meant that mac os builds could pick up the wrong header
fSeries.hh is renamed to f_series.hh

erikvonreis committed:
More changes to git mac working

erikvonreis committed:
MNT: Re-rendered with conda-build 25.7.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.08.08.10.31.19

erikvonreis committed:
MNT: Re-rendered with conda-build 25.7.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.08.08.10.31.19

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

erikvonreis committed:
Fix tests

erikvonreis committed:
MNT: Re-rendered with conda-build 25.7.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.08.08.10.31.19
